mb002270/aws-servicebroker
# Fixed typo in documentation

Changed "Currentently" to "Currently" in the docs/readme.md file



## Overview

Brief description of what this PR does, and why it is needed (use case)?
Typo fix in README file only.

## Related Issues

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #
No bug report filed.  No active PRs for this.  Just a typo.

## Testing

How did you validate the changes in this PR? If there are unit tests included describe what they test
Text only change, fix to a single word.  Viewed Readme to verify fix.

### Notes

Optional. Caveats, Alternatives, Other relevant information.

## Testing Instructions

 How to test this PR Start after checking out this branch (bulleted)
 * Include test case, and expected output
Visually verify "Currentently" is now spelled "Currently"

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
